book/src/build_and_install.md

```diff
@@ -21,22 +42,8 @@ major version of the `llvm-sys` crate. Currently you will need to install LLVM 1
 the appropriate environment variable (`LLVM_SYS_130_PREFIX=/usr/lib/llvm-13` for LLVM 13), so
 the build of the `llvm-sys` crate knows what files to grab.
 
-## Cloning the repository
-On your local computer, open up a shell and clone the repository.
-```bash
-git clone https://github.com/PLC-lang/rusty
-cd rusty
-```
-
 ## Building
-Building is as easy as typing a single command. If you just want to use the
-compiler without doing development, building the release configuration will
-give you a smaller and likely a faster binary.
-```bash
-cargo build --release
-```
-
-You can find the binary at `./target/release/rustyc`.
+Just like any Rust project binaries can be built with `cargo build`. For release builds, i.e. faster and smaller binaries, you have to pass the `--release` flag, like so `cargo build --release`. The resulting binaries can be found at `target/release/rusty`
```

Comment:
rustyc is the bin name, rusty is just the lib name

Reply:
Fixed, thanks :)